test/test_sparse.py

```diff
@@ -1524,11 +1580,15 @@ def test_shape(di, dj, dk, nnz):
         true_result = (bias.to_dense() + torch.matmul(weight.to_dense(), x)).to_sparse()
         self.assertEqual(self.safeToDense(res), self.safeToDense(true_result))
 
+    @unittest.skipIf(TEST_WITH_CROSSREF, "generator unsupport triggers assertion error")
```

Comment:
Could you file an issue for this?

Reply:
Done: https://github.com/pytorch/pytorch/issues/97284